client/app/dedicated/server/interfaces/ola-step-checker/ola-step-checker.controller.js

```diff
@@ -0,0 +1,46 @@
+import _ from 'lodash';
+
+export default class {
+  /* @ngInject */
+  constructor($location, $state) {
+    this.$location = $location;
```

Comment:
Seems not used ?

Reply:
Is used in the commented (for now) part of `closeSteps()`